examples/ivr/dialer.py

```diff
@@ -80,16 +80,20 @@ def create_url():
         # Adding F at the end of the DID disables remote SS7 overlap dialing
         # which can add 5 sec to the incoming call setup time.
         return {
-            'dest_url': 'a/4113',
+            'dest_url': 'a/1000F',
             'dest_profile': 'g1',
-            'dest_endpoint': 'freetdm'
+            'dest_endpoint': 'freetdm',
+            'caller_id': '\'Sangoma Technologies\'',
```

Comment:
Yeah doesn't really matter. `\` is pretty hideous. If you really want to use all the same quoting that's fine but I would still remove the backslashes. See pep8:
https://www.python.org/dev/peps/pep-0008/#string-quotes